styles.css

```diff
@@ -0,0 +1,375 @@
+    top: 2rem;
+}
+.sign-up p {
+    left: 12.3%;
+    position: relative;
+    top: 2.7rem;
+}
+.news-logo {
+    background-color: #f72210;
+    color: white;
+    float: left;
+    font-weight: bold;
+    font-size: 1.2rem;
+    left: 1%;
+    position: relative;
+    text-align: center;
+    top: 1.7rem;
+    width: 10%;
+}
+.news-img {width: 100%;}
+
+/*  "In The Magazine" & "Editor's Pick" Styles */
+
+.trp-img {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 16rem;
+}
+.mag-img {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 16rem;
+}
+.m1 {background-image: url(/Images/august-cover.jpg);}
+.m2 {background-image: url(/Images/bondage.jpg);}
+.m3 {background-image: url(/Images/terry.jpg);}
+.m4 {background-image: url(/Images/divided.jpg);}
+.m5 {background-image: url(/Images/money.jpg);}
+.m6 {background-image: url(/Images/strawberries.jpg);}
+.m7 {background-image: url(/Images/rupaul.jpg);}
+.m8 {background-image: url(/Images/world-class-relief.jpg);}
+.second-section article h6 {
+    font-size: 1.1rem;
+    font-weight: bold;
+}
+.category-1 {
+    background-color: #f72210;
+    color: white;
+}
+.category-2 {
+    background-color: white;
+    color: #f72210;
+}
+.trp-1 {
+    height: 22.5rem;
+}
+.trp-2 {
+    background-color: #E8E6E6;
+    height: 2.9rem;
+}
+.trp-2 p {
+    font-weight: bold;
+    position: relative;
+    text-align: center;
+    bottom: -20%;
+}
+.section2-div2 article{
+    box-sizing: border-box;
+    height: auto;
+    margin-left: -0.5rem;
+    margin-right: -0.5rem;
+}
+.mag-heading {
+    margin: 0 auto;
+    margin-bottom: 2%;
+    text-align: center;
+}
+.mag-heading span {
+    font-size: 1rem;
+    font-weight: bold;
+}
+.news-logo-2 {
+    background-color: #f72210;
+    color: white;
+    float: left;
+    font-weight: bold;
+    font-size: 1.2rem;
+    left: 41%;
+    position: relative;
+    text-align: center;
+}
+.magazine {
+    left: 2%;
+    position: relative;
+}
+.m9 {background-image: url(/Images/visit.jpg);}
+.m10 {background-image: url(/Images/trump-taxes.jpg);}
+.m11 {background-image: url(/Images/home-alone.jpg);}
+.m12 {background-image: url(/Images/toni.jpg);}
+.m13 {background-image: url(/Images/horizon-2.jpg);}
+.m14 {background-image: url(/Images/beto-orourke.jpg);}
+.m15 {background-image: url(/Images/vegan.jpg);}
+.m16 {background-image: url(/Images/soldier.jpg);}
+.fourth-section h6 {
+    font-size: 1.2rem;
+    font-weight: bold;
+}
+
+/*  Featured Slideshows Section Styles */
+
+.fifth-section h4 {
+    color: black;
+    font-weight: bold;
+    text-align: center;
+}
+.divider {
+    color: #E0E0E0;
+    width: 92%;
+}
+.sixth-section {
+    margin-top: 0.5rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+}
+.sixth-section h4 {
+    color: #f72210;
+    font-weight: bold;
+}
+.m17 {background-image: url(/Images/manson.jpg);}
```

Comment:
Your images are not showing up instead of using (/Images/rocket.jpg)  a  `.` before the slash like this `./`